### PR TITLE
Refactor approvals

### PR DESCRIPTION
**Pull request by sponkrashin**

### CODE REVIEW

File: server/Arcadia.Assistant.Organization/EmployeeActor.cs [hunk 1]
```diff
@@ -47,16 +46,11 @@ public class EmployeeActor : UntypedPersistentActor, ILogReceive
             var vacationActorProps = EmployeeVacationsActor.CreateProps(
                 this.employeeMetadata.EmployeeId,
                 this.employeeFeed,
-                vacationsRegistry,
-                calendarEventsApprovalsChecker);
+                vacationsCreditRegistry,
+                null);
```
Comment (santee) [inline server/Arcadia.Assistant.Organization/EmployeeActor.cs]: null?

Comment (santee) [reply server/Arcadia.Assistant.Organization/EmployeeActor.cs]: maybe delete it altogether?

Comment (sponkrashin) [reply server/Arcadia.Assistant.Organization/EmployeeActor.cs]: Accidentally missed.

Comment (sponkrashin) [reply server/Arcadia.Assistant.Organization/EmployeeActor.cs]: Fixed.

**Pull request merged**